File: proyecto_java/proyecto_final/src/paquete07/problemaYoga.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete07;

import java.util.Scanner;

/**
 *
 * @author utpl
 */
public class ProblemaYoga {

    public static String obtenerDatosYoga() {

        String cadena;

        Scanner entrada = new Scanner(System.in);
        System.out.println("Ingrese nombre del participante");
        String nombrePersona = entrada.nextLine();

        System.out.println("Ingrese su edad");
        int edad = entrada.nextInt();

        System.out.println("Ingrese su nivel"
                + "(principiante, intermedio, avanzado)");
        entrada.nextLine();
        String nivel = entrada.nextLine();

        System.out.println("Ingrese su estilo de yoga favorito");
        String estilo = entrada.nextLine();

        System.out.println("Ingrese su ciudad de residencia");
        String ciudad = entrada.nextLine();

        cadena = String.format("Los datos ingresados para yoga son: "
                + "\nNombre: %s\nEdad: %d\nCiudad: %s\nNivel: %s\n"
                + "Estilo favorito: %s\n",
                nombrePersona, edad, ciudad, nivel, estilo);

        return cadena;

    }
}

File: proyecto_python/proyectofinal2b/problemaYoga.py
def obtenerDatosYoga():
    print("Ingrese el nombre del participante:")
    nombre = input()

    print("Ingrese la edad del participante:")
    edad = input()
    edad = float(edad)

    print("Ingrese su nivel:"
                + "(principiante, intermedio, avanzado)")
    nivel = input()

    print("Ingrese su ciudad de residencia")
    ciudad = input()

    print("Ingrese su estilo de yoga favorito")
    estilo = input()

    cadena = (f"\nLos datos ingresados son: "
              f"\nNombre: {nombre}\nEdad: {edad}\nNivel: {nivel}"
              f"\nCiudad de residencia: {ciudad}\nEstilo favorito: {estilo}\n")

    return cadena